rest-api/scripts/setup-local.sh

```
@@ -221,32 +226,70 @@ create_site() {
-    local reg_token="${SITE_REG_TOKEN:-local-dev-otp}"
     local sm_cacert
     sm_cacert=$(kubectl -n $NAMESPACE get secret site-manager-tls -o jsonpath='{.data.ca\.crt}' 2>/dev/null | base64 -d 2>/dev/null || echo "")
 
-    kubectl -n $NAMESPACE delete secret site-registration 2>/dev/null || true
-    kubectl -n $NAMESPACE create secret generic site-registration \
-        --from-literal=site-uuid="$site_id" \
-        --from-literal=otp="$reg_token" \
-        --from-literal=creds-url="https://nico-rest-site-manager:8100/v1/sitecreds" \
-        --from-literal=cacert="$sm_cacert"
+    if [ -n "${SITE_REG_TOKEN:-}" ] && [ "$SITE_REG_TOKEN" != "null" ]; then
+        kubectl -n $NAMESPACE create secret generic site-registration \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)